src/main/java/seedu/address/model/person/Patient.java

```diff
@@ -0,0 +1,36 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+/**
+ * Represents a Patient in ORGANice.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Patient extends Person {
+    //fields
+    private final Age age;
+
+    /**
+     * Every field must be present and not null.
+     */
+    public Patient(Type type, Nric nric, Name name, Phone phone, Age age) {
+        super(type, nric, name, phone);
+        requireAllNonNull(age);
+        this.age = age;
```

Comment:
why only `requireAllNonNull` for age?

Reply:
This is because in the base class's constructor, the requireAllNonNull() method was applied for type, nric, name, and phone. So the only attribute we need to apply this method too is the age attribute.